### PR TITLE
Member filter

### PR DESCRIPTION
**Pull request by YunzheZJU**
This pr implements #4, allowing users to filter members to be displayed.

#### Preview
options

#### TODO
- [x] Translation for Japanese (help wanted!)

### CODE REVIEW

File: src/popup/i18n/locales/ja.json5 [hunk 1]
```diff
@@ -36,6 +36,8 @@
           "zh-CN": "简体中文",
         },
       },
+      // Prompt users to change advanced settings
+      advanced: "Advanced Settings",
```
Comment (YunzheZJU) [inline src/popup/i18n/locales/ja.json5:40]:
```suggestion
      advanced: "TODO:Advanced Settings",
```

**Review by ReiRokusanami0010 — COMMENTED** [review]
I tried to translate it.  
Please review.

File: src/options/i18n/locales/ja.json5 [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  // Component App
+  app: {
+    // Advanced settings
+    advancedSettings: {
+      // Label for advancedSettings
+      label: "TODO:Advanced Settings",
```
Comment (ReiRokusanami0010) [inline src/options/i18n/locales/ja.json5:7]:
```suggestion
      label: "詳細設定",
```

File: src/options/i18n/locales/ja.json5 [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      // Prompt users to configure the list of members to subscribe
+      members: {
+        // Label for members
+        label: "TODO:Members",
```
Comment (ReiRokusanami0010) [inline src/options/i18n/locales/ja.json5:11]:
```suggestion
        label: "メンバー一覧",
```

File: src/options/i18n/locales/ja.json5 [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        // Description for members
+        description: "TODO:Configure the list of members to subscribe"
```
Comment (ReiRokusanami0010) [inline src/options/i18n/locales/ja.json5:13]:
```suggestion
        description: "登録するメンバーを設定してください"
```

File: src/options/i18n/locales/ja.json5 [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      }
+    }
+  },
+  // Component MemberGroup
+  memberGroup: {
+    // Indicate that no member data can be found for this group
+    noData: "TODO:No Member Data",
```
Comment (ReiRokusanami0010) [inline src/options/i18n/locales/ja.json5:20]:
```suggestion
    noData: "メンバーのデータがありません。",
```

File: src/popup/i18n/locales/ja.json5 (hunk 1 above)
Comment (ReiRokusanami0010) [reply src/popup/i18n/locales/ja.json5:40]:
```suggestion
      advanced: "詳細設定",
```

**Review by ReiRokusanami0010 — COMMENTED** [review]
I didn't save the translation and replied to it, so I reviewed it again.

File: src/options/i18n/locales/ja.json5 (hunk 5 above)
Comment (YunzheZJU) [reply src/options/i18n/locales/ja.json5:20]: Let me remove the `。`

Comment (YunzheZJU) [reply src/options/i18n/locales/ja.json5:20]:
```suggestion
    noData: "メンバーのデータがありません",
```

**Comment by YunzheZJU** [comment]
> I didn't save the translation and replied to it, so I reviewed it again.

お疲れ様でした。ありがとうございます！

**Pull request merged**